src/pkg/packager/deploy.go

```diff
@@ -290,7 +291,7 @@ func (d *deployer) deployInitComponent(ctx context.Context, pkgLayout *layout.Pa
 
 	// Before deploying the seed registry, start the injector
 	if isSeedRegistry {
-		err := d.c.StartInjection(ctx, pkgLayout.DirPath(), pkgLayout.GetImageDirPath(), component.Images, d.s.RegistryInfo.NodePort)
+		err := d.c.StartInjection(ctx, pkgLayout.DirPath(), pkgLayout.GetImageDirPath(), component.Images, d.s.RegistryInfo.NodePort, &pkgLayout.Pkg)
```

Comment:
I'd lean towards skipping the labeling of the injector. I can see the argument either way on if it'd be accurate, on one hand there is an injector component in the init package, on the other hand these resources are created by Zarf itself. Given that the injector is running for ~15 seconds, I'd rather air on keeping our code simpler unless there is a use case for it.

Reply:
Fair enough